packages/prisma/migrations/20220525182228_cal_video_preinstalled/migration.sql

```diff
@@ -0,0 +1 @@
+insert into "App" ("slug", "dirName", "updatedAt", "categories") values ('daily-video', 'dailyvideo', NOW(), '{video}')
```

Comment:
This migration is breaking builds. Removed `autoupdate`. Let's abstain to push until this is resolved.